tests/test_frontend/test_view_a_licence/test_templates/test_view_a_license_application.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from django.urls import reverse
+from django.utils import timezone
+from playwright.sync_api import expect
+
+from tests.factories import LicenceFactory, UserFactory
+from tests.test_frontend.conftest import PlaywrightTestBase
```

Comment:
you need to use the specific view-a-licence `PlaywrightTestBase` so it goes to the correct URL:

```suggestion
from tests.test_frontend.test_view_a_licence.conftest import PlaywrightTestBase
```